src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java

```diff
@@ -667,7 +667,7 @@ private void addBackButton(ChestMenu menu, int slot, Player p, PlayerProfile pro
     private void displayRecipes(Player p, PlayerProfile profile, ChestMenu menu, RecipeDisplayItem sfItem, int page) {
         List<ItemStack> recipes = sfItem.getDisplayRecipes();
 
-        if (!recipes.isEmpty()) {
+        if (recipes != null && !recipes.isEmpty()) {
```

Comment:
This is marked as non-null.

Reply:
This wasnt supposed to be in here whoops